helm_deploy/prison-offender-events/templates/retry-dlqs-cronjob.yaml

```diff
@@ -1,12 +1,11 @@
 apiVersion: batch/v1
 kind: CronJob
 metadata:
-  name: {{ include "app.fullname" . }}-queue-housekeeping-cronjob
+  name: {{ include "app.fullname" . }}--retry-dlqs
```

Comment:
took opportunity to rename job as only housekeeping it is doing is retry on the DLQs